docs/source/faqs.rst

```diff
@@ -180,23 +182,24 @@ board's static IP here.
 How do I find my hostname?
 --------------------------
 
-Connect to the board using a terminal and run ``hostname``
+Connect to the board using a terminal and run:
+
+.. code-block:: console
+   
+      hostname
    
 How do I change the hostname?
 -----------------------------
 
 If you have multiple boards on the same network, you should give them different
-host names.  You can change the hostname by editing the Linux hostname file:
-
-   .. code-block:: console
-   
-      /etc/hostname
+host names.  You can change the hostname by editing the Linux hostname files:
+``/etc/hostname`` and ``/etc/hosts``.
    
 What is the user account and password?
 --------------------------------------
 
-Username and password for all Linux, jupyter and samba logins are:
-``xilinx/xilinx``
+Username and password for all Linux, Jupyter and Samba logins are:
```

Comment:
I split this in Getting started - can you split into somethin like : The username for Jupyter and Samba is **xilinx** and the password is **xilinx**?

Just in case someone gets confused and thinks the username is xilinx/xilinx